Create Moderation.md (#25831)
microsoft/winget-pkgs: Moderation.md

# Moderation Overview

The Windows Package Manager community repository is the location for manifests published in the default Windows Package Manager source. Automated pipelines validate and publish these manifests. The quality of the metadata has a direct impact on customer experience. Human review has proven to be an invaluable tool for ensuring the best experiences can be provided.

## Moderators

> In addition to Microsoft employees, several community members have been identified through their high quality submissions, willingness to help others, and adhere to our [code of conduct](CODE_OF_CONDUCT.md). You can see who they are in a pinned [public service announcement](https://github.com/microsoft/winget-pkgs/issues/15674).

Reviewer: ```suggestion
In addition to Microsoft employees, several community members have been identified through their high quality submissions, willingness to help others, and adherence to our [code of conduct](CODE_OF_CONDUCT.md). You can see who they are in a pinned [public service announcement](https://github.com/microsoft/winget-pkgs/issues/15674).
```

When moderation was initially implemented, several objective criteria were observed. These included the number of Pull Requests (PR)s made, the length of time they had been active in the project, and their interaction with others who submitted PRs.

> Our intent was not to introduce a numbers game for others to achieve and automatically become a moderator. Our intent going forward is to ensure the community is sufficnetly supported with individuals who have the technical knowledge and a proven track record of success. 

Reviewer 2: Just a minor spelling mistake I noticed when reading through it.

```suggestion
Our intent was not to introduce a numbers game for others to achieve and automatically become a moderator. Our intent going forward is to ensure the community is sufficiently supported by individuals who have the technical knowledge and a proven track record of success. 
```

## Expectations

> Moderators are expected to continue behaving in a manner consistent with what led to their nomination. In addition, they are given the ability to approve PRs for manifests. This should not be seen as the goal, however. The goal is to help ensure high quality manifests, and to help the community with package submission. They should also feel free to take time off and relax. At any time and for any reason they may request to discontinue this responsibility, and it will be honored.

Reviewer: ```suggestion
Moderators are expected to continue behaving in a manner consistent with what led to their nomination. In addition, they are given the ability to approve PRs for manifests. This should not be seen as the goal, however. The goal is to help ensure high quality manifests and to help the community with package submission. They should also feel free to take time off and relax. At any time and for any reason they may request to discontinue this responsibility, and it will be honored.
```

> Our intent was not to introduce a numbers game for others to achieve and automatically become a moderator. Our goal from now on is to ensure the community is sufficiently supported by individuals who have the technical knowledge and a proven track record of success. Becoming a moderator is like becoming a [Microsoft Most Valuable Professional](https://mvp.microsoft.com/Pages/what-it-takes-to-be-an-mvp). There isn't a set formula. We're looking at what individual contributors are doing, and how they are doing it. If the need arises to add additional moderators, nominations may come from one of the core team members or an existing moderator. We will publicly disclose the nomination in a discussion. If the nominee agrees to the requirements in this document, they may be awarded the status of moderator.

Reviewer 3: ```suggestion
Our intent was not to introduce a numbers game for others to achieve and automatically become a moderator. Our goal from now on is to ensure the community is sufficiently supported by individuals who have the technical knowledge and a proven track record of success. Becoming a moderator is like becoming a [Microsoft Most Valuable Professional](https://mvp.microsoft.com/en-us/Pages/what-it-takes-to-be-an-mvp). There isn't a set formula. We're looking at what individual contributors are doing, and how they are doing it. If the need arises to add additional moderators, nominations may come from one of the core team members or an existing moderator. We will publicly disclose the nomination in a discussion. If the nominee agrees to the requirements in this document, they may be awarded the status of moderator.
```
Looks like this page is broken for me without the `en-us`:
![image](https://user-images.githubusercontent.com/11600822/131386677-bf4b8748-39b2-45bf-92f6-58b9e5f1e0dd.png)

I'm in Europe (United Kingdom - en-GB). Not sure if anyone else is running into this.

Reviewer 2: I'm receiving the same error. Seems to only be available in following languages:

![image](https://user-images.githubusercontent.com/52826317/131387076-9e0ac885-8ca1-4d27-982d-b50670d3db98.png)

Reviewer 2: It still errors even if you select one of those languages 🤦‍♂️

Reviewer 4: Note the `ru-RU` in URL
![image](https://user-images.githubusercontent.com/83997633/131389932-45132053-1190-4261-b788-d832f3bab7ef.png)

Reviewer 4: Works with `de-DE` though...
![image](https://user-images.githubusercontent.com/83997633/131390154-e1982def-41e0-4054-a9ae-258d04cf6bff.png)

Reviewer 2: > Works with `de-DE` though...

`de-DE` is Deutsch/German which is listed on the website. 

The languages themselves work, but the URL without the language codes does not redirect/display correctly (if it's even supposed to, I would assume so as some of the URLs does display the correct language that you select without needing the language codes in the URL). Seems to only be an issue with some URLs on their website.

`en-US` is obviously the best option here like @ItzLevvie suggested. Ideally, the website should be updated and follow the same style (or similar) across the board as Microsoft Docs where you get redirected to `en-US` if the language is not listed, or you get redirected to the language you have already selected on that site.

### Reviewing Pull Requests

> Moderators should review PRs to make sure the metadata is accurate, and that the Windows Package Manager will behave in a predictable manner with the given manifest. This includes reviewing the metadata and testing the installation of packages. Ideally, common issues will be documented, and referred to for the sake of consistency. This might also include tips and tooling to help with the process. Some users are new to GitHub, and may need a bit more support. We've all been new to Git at one point.

Reviewer 5: > We've all been new to Git at one point.

https://xkcd.com/1597/

Reviewer: ```suggestion
Moderators should review PRs to make sure the metadata is accurate and that the Windows Package Manager will behave in a predictable manner with the given manifest. This includes reviewing the metadata and testing the installation of packages. Ideally, common issues will be documented and referred to for the sake of consistency. This might also include tips and tooling to help with the process. Some users are new to GitHub and may need a bit more support. We've all been new to Git at one point.
```

### Providing Feedback

> Moderators are often on the front line when new issues are identified. They should collaborate with each other, the community, and the product team. This will help ensure the Windows Package Manager continues to improve. Sometimes this could be by creating an Issue or a Discussion. Other times this may just be discussion in a PR.

Reviewer: It seems like an ending statement is needed here. Maybe something about being respectful and understanding, or adding value with their feedback?

Author: Thanks for the feedback 😊.
